nuget/System.Net.Libuv.nuspec

```diff
@@ -20,6 +20,7 @@
   </metadata>
   <files>
     <file src="..\bin\Windows_NT.AnyCPU.Release\System.Net.Libuv\System.Net.Libuv.dll" target="lib/portable-net45+win8+wpa81+aspnetcore50"/>
+    <file src="..\bin\Windows_NT.AnyCPU.Release\System.Net.Libuv\libuv.dll" target="runtimes/win7-x64/native"/>
```

Comment:
I think @davidfowl  just meant to mimic the structure of their package, which is a fine suggestion.  Have a look at their kestrel package.  If size is small you can ignore my split suggestion (which in retrospect is probably a bad idea since NuGet only follows runtime.json in the compile graph).

Here is the package @davidfowl was referring to: https://www.nuget.org/api/v2/package/Microsoft.AspNet.Server.Kestrel/1.0.0-rc1-final